signalfx/resource_signalfx_team_links.go

```diff
@@ -0,0 +1,183 @@
+package signalfx
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func teamLinksResource() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"team": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Team ID to link the configured dashboard groups or detectos to.",
+			},
+			"dashboard_groups": {
+				Type:        schema.TypeSet,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "Dashboard group IDs to link to the team.",
+				Optional:    true,
+			},
+			"detectors": {
+				Type:        schema.TypeSet,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "Detector IDs to link to the team.",
+				Optional:    true,
+			},
+		},
+		Create: teamLinksCreate,
+		Read:   teamLinksRead,
+		Update: teamLinksUpdate,
+		Delete: teamLinksDelete,
+		Exists: teamLinksExists,
+	}
+}
+
+func teamLinksExists(d *schema.ResourceData, meta interface{}) (bool, error) {
+	return true, nil
```

Comment:
Is this always true because the links are such an abstract concept or what?

Reply:
Right since there's no actual resource on the backend that gets created or deleted this value doesn't really matter. More details here https://www.terraform.io/docs/plugins/provider.html#resources

Actually looks like it's optional so I can just not even implement it since it has no use. Will remove it.